src/test/java/nextstep/ladder/domain/ParticipantsTest.java

```diff
@@ -0,0 +1,44 @@
+package nextstep.ladder.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ParticipantsTest {
+
+    private List<String> expected;
+    private String input;
+
+    @BeforeEach
+    void setUp() {
+        expected = Arrays.asList("pobi", "honux", "crong", "jk");
+        input = String.join(",", expected);
+    }
+
+    @Test
+    @DisplayName("참여자는 사용자 이름을 반환한다.")
+    void test01() {
+        Participants participants = Participants.of(input);
+        List<String> actual = participants.getUserNames()
+                .stream()
+                .map(UserName::getName)
+                .collect(Collectors.toUnmodifiableList());
+
+        assertThat(actual).isEqualTo(expected);
```

Comment:
```suggestion
       List<UserName> actual = participants.getUserNames();
       혹은,
       
       // List<String> 타입을 반환하는게 실질적으로 필요했던게 아닐까요? 
       List<String> actual = participants.getUserNames();

        assertThat(actual).isEqualTo(expected);
```

테스트코드가 복잡하면서, 너무 많은 것을 알려고 하는것 같아요.
테스트 하고자 하는 메소드에 집중하면 좋을듯해요!